pyproject.toml

```diff
@@ -157,9 +157,8 @@ dependencies = [
   "isort==5.13.2",
   "check-sdist==0.1.3",
   "sqlalchemy[mypy]==1.4.41",
-  "yamlfix==1.16.1",
-  # because of aleph messages otherwise yamlfix install a too new version
-  "pydantic>=1.10.5,<2.0.0",
+  "yamlfix>=1.17",
```

Comment:
We probably want to cap this version at some point to avoid having random breaking situations

Reply:
So In that case I will change to only upgrade the patches versions of the library to avoid future issues.